### ISSUE TITLE
Warframe fails to lauch due to missing DirectX 9

### ISSUE THREAD

**Issue by dyc3** [GloriousEggroll/warframe-linux#53]
After the launcher script finishes downloading updates, a black window appears, with the path to `Warframe.exe` as the window title, and almost immediately closes. Finally, an error message box appears on screen with the message "DirectX 9 was not found on your system."
![2018-05-02-182531_334x159_scrot](https://user-images.githubusercontent.com/1808807/39552404-1ddc0db6-4e37-11e8-989f-d36b2b924c4e.png)

**Comment by GloriousEggroll** [comment]
did you change anything on the launch script or install script? because by default the game launches in dx11 mode, not dx9

**Comment by dyc3** [comment]
No, the launch script and install script are unmodified. However, the WINEPREFIX is the same as the previous version of warframe I had working
